src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/service/impl/FlowServiceImpl.java

```diff
@@ -672,6 +675,28 @@ private FlowPathPayload buildFlowPathPayload(List<FlowPathDto> paths, String flo
                         .collect(Collectors.toList()));
     }
 
+    @Override
+    public CompletableFuture<Map<Instant, String>> getFlowStatusEvents(String flowId, long timestampFrom,
```

Comment:
why do you add a new endpoint to old v1 instead of v2?

Reply:
Moved to v2.